app/models/concerns/orchestration/dhcp.rb

```diff
@@ -87,11 +87,8 @@ def boot_server
     bs = tftp.bootServer
     # if that failed, trying to guess out tftp next server based on the smart proxy hostname
     bs ||= URI.parse(subnet.tftp.url).host
-    # now convert it into an ip address (see http://theforeman.org/issues/show/1381)
-    ip = NicIpResolver.new(:nic => self).to_ip_address(bs) if bs.present?
```

Comment:
This is the only use of `NicIpResolver`. Let's drop it if we no longer use it.

Reply:
Will gladly drop that.